sailfishusermanager.cpp

```diff
@@ -535,3 +536,80 @@ void SailfishUserManager::updateEnvironment(uint uid)
         file.close();
     }
 }
+
+QStringList SailfishUserManager::usersGroups(uint uid)
+{
+    m_exitTimer->start();
+    return m_lu->groups(uid);
+}
+
+void SailfishUserManager::addToGroups(uint uid, const QStringList &groups)
+{
+    m_exitTimer->start();
```

Comment:
I wonder if this should be before or after checkAccessRights. At least it should be consistent so either change this or change the others.

Reply:
Moved